tests/integration/ceph_base_nfs_test.go

```diff
@@ -42,17 +42,29 @@ func runNFSFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suit
 	if settings.TestNFSCSI {
 		// Following two commands are needed to be able to create NFS exports in ceph v17.2
 		// refer: https://github.com/rook/rook/blob/master/Documentation/CRDs/ceph-nfs-crd.md#ceph-v1720
-		parameters := []string{"orch", "set", "backend"}
 		clusterInfo := client.AdminTestClusterInfo(settings.Namespace)
+		parameters := []string{"mgr", "module", "enable", "rook"}
 		cmd, args := client.FinalizeCephCommandArgs("ceph", clusterInfo, parameters, k8sh.MakeContext().ConfigDir)
 		res, err := k8sh.MakeContext().Executor.ExecuteCommandWithOutput(cmd, args...)
+		logger.Info(res)
 		if err != nil {
 			logger.Errorf("Error executing command %q: <%v>, %q", parameters, err, res)
 			assert.NoError(s.T(), err)
 		}
-		parameters = []string{"mgr", "module", "disable", "rook"}
+
+		parameters = []string{"mgr", "module", "enable", "nfs"}
+		cmd, args = client.FinalizeCephCommandArgs("ceph", clusterInfo, parameters, k8sh.MakeContext().ConfigDir)
+		res, err = k8sh.MakeContext().Executor.ExecuteCommandWithOutput(cmd, args...)
+		logger.Info(res)
+		if err != nil {
+			logger.Errorf("Error executing command %q: <%v>, %q", parameters, err, res)
+			assert.NoError(s.T(), err)
+		}
+
+		parameters = []string{"orch", "set", "backend", "rook"}
```

Comment:
The rook backend with NFS is broken for 17.2.0, and I'm not sure if it's fixed for 17.2.1. We might have to enable this for v16 and disable it for v17.